data-table/table.js

```diff
@@ -25,9 +25,9 @@ class Config {
      */
     constructor(clientConfig) {
         this.numRows = clientConfig.numRows === undefined ? 3 : clientConfig.numRows;
-        this.numColumns = clientConfig.numColumns === undefined ? 4 : clientConfig.numColumns;
-        this.rowsName = clientConfig.rowsName === undefined ? "row" : clientConfig.rowsName;
-        this.columnsName = clientConfig.columnsName === undefined ? "column" : clientConfig.columnsName;
+        this.numColumns = clientConfig.numColumns === undefined ? 4 : clientConfig.numColumns + 1;
```

Comment:
Because the column that contains the name for that row (candidate names, etc.) is included in that number. I'm making the assumption that the when the user says they want a table with three columns, what they really mean is "three columns plus a column for row names".

Added a comment explaining that logic! I'm happy to revisit this, and even change the code if we think that's best, but for now I'm marking this resolved.